.nvmrc

```diff
@@ -1 +1 @@
-v16.12.0
+18
```

Comment:
Why don't you mention here `v18.18` instead of `18` ? 

I had previously node 18.16 installed locally, so first it logged : 

```
~/Dropbox/_CODE/_MULTI/betagouv/transition-ecologique-entreprises-widget  (feature/update-node) $
nvm use
Found '/Users/jpy/Dropbox/_CODE/_MULTI/betagouv/transition-ecologique-entreprises-widget/.nvmrc' with version <18>
Now using node v18.16.0 (npm v9.5.1)

🌱 jpy 23-10-16 17:52
~/Dropbox/_CODE/_MULTI/betagouv/transition-ecologique-entreprises-widget  (feature/update-node) $
npm i
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'tee@1.0.0',
npm WARN EBADENGINE   required: { node: '~18.18', npm: '~9.8' },
npm WARN EBADENGINE   current: { node: 'v18.16.0', npm: '9.5.1' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@tee/backend@1.0.0',
npm WARN EBADENGINE   required: { node: '~18.18', npm: '~9.8' },
npm WARN EBADENGINE   current: { node: 'v18.16.0', npm: '9.5.1' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@tee/web@0.1.0',
npm WARN EBADENGINE   required: { node: '~18.18', npm: '~9.8' },
npm WARN EBADENGINE   current: { node: 'v18.16.0', npm: '9.5.1' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@tee/backend@1.0.0',
npm WARN EBADENGINE   required: { node: '~18.18', npm: '~9.8' },
npm WARN EBADENGINE   current: { node: 'v18.16.0', npm: '9.5.1' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@tee/web@0.1.0',
npm WARN EBADENGINE   required: { node: '~18.18', npm: '~9.8' },
npm WARN EBADENGINE   current: { node: 'v18.16.0', npm: '9.5.1' }
npm WARN EBADENGINE }
```

Reply:
Once I changed `.nvmrc` content to `v18.18` these errors are not appearing anymore.

Reply:
Oh, you right, good catch !!

I have change on all packages.json with:
```json
  "engines": {
    "node": "18.x",
    "npm": "9.x"
  },
```
to accept all minors and patches versions